LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/Processors/LnsProtocolMessageProcessor.cs

```diff
@@ -202,11 +196,6 @@ internal async Task InternalHandleDataAsync(RouteValueDictionary routeValues, We
                     {
                         var jreq = LnsData.JoinRequestFrameReader.Read(json);
 
-                        if (this.joinRequestDeduplication.ShouldDrop(jreq, stationEui))
-                        {
-                            break;
-                        }
-
                         this.joinRequestCounter?.Add(1);
```

Comment:
[nit] please pay attention that the semantics of that metric does not change (we only count the number of join requests after dedup)

Reply:
Since the deduplication is moved, we would need to move this later then.

Reply:
Fixed with fc67d8b8bae90890ac4bba8569a57a53ec0ea7be